lib/main.js

```diff
@@ -27,6 +27,9 @@ Error.stackTraceLimit = Infinity;
 
 function test(props) {
 	if (isFailed) {
+		if (props.pending) {
+			adapter.send('test', props);
+		}
```

Comment:
This only applies to when `--fail-fast` is used. I don't think we don't need to print pending tests in that circumstance.

Reply:
I imagine people want to be able to identify _any_ non-passing tests with `--fail-fast`. That's doable without this PR if all tests are serial (--> any tests after the failed one are pending), but harder to reason about if test order is nondeterministic or involves multiple files executing concurrently.

Thoughts?

Reply:
> I imagine people want to be able to identify any non-passing tests with --fail-fast. 

No, all tests should stop running as soon as a single failure occurs. See #1158.

Reply:
> iterative development/debugging. Personally, I'd find knowing which tests are pending useful in this case - especially if I'm trying to troubleshoot why said tests are pending.

It would show tests that haven't yet completed when another test fails. I don't think that's what we're trying to solve with this PR.